### PR TITLE
add test2.py

### PR DESCRIPTION
**Pull request by slayerzeroa**
add test2.py for test

### CODE REVIEW

**Comment by slayerzeroa** [comment]
I accept iy

**Review by slayerzeroa — COMMENTED** [review]
I accept it.

File: test.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Hello, World!")
```
Comment (slayerzeroa) [inline test.py:1]: ok

File: test2.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("Hi, Hello, World!")
```
Comment (slayerzeroa) [inline test2.py:1]: ok

**Review by psyduck-01 — APPROVED** [review]
I approve it

**Pull request merged**